examples/rugby_ball/objectives/volume_constraint.f90

```diff
@@ -200,13 +200,15 @@ subroutine volume_constraint_compute_sensitivity(this, design, fluid, adjoint)
     class(adjoint_scheme_t), intent(in) :: adjoint
 
     call field_rone(this%sensitivity_to_coefficient)
-    call field_cmult(this%sensitivity_to_coefficient, 1.0_rp/fluid%c_xh%volume)
 
     if(this%min_max) then
```

Comment:
please change the name of this variable. It is impossible to know what it does.

My suggestion would be `is_min` or `is_max`